universal-login-react/src/ui/TopUp/Fiat/FiatFooter.tsx

```diff
@@ -48,17 +54,17 @@ export const FiatFooter = ({paymentMethod, walletService, selectedCurrency}: Fia
             <InfoText>You have to install Yoti mobile app</InfoText>
             <img src={Yoti} srcSet={Yoti2x} className="yoti-logo" alt="Yoti" />
           </div>
-          <div className="info-block info-row">
-            <InfoText>Minimum amount is {minimumAmount}€</InfoText>
-          </div>
+          {minimumAmounts && <div className="info-block info-row">
+            <InfoText>Minimum amount is {minimumAmounts.generalMinimalAmount}€</InfoText>
+          </div>}
```

Comment:
Is `minimumAmounts && ` intended? This code should not change anything compared to our previous code:
```ts
{div className="info-block info-row">
    <InfoText>Minimum amount is {minimumAmounts?.generalMinimalAmount}€</InfoText>
</div>}
```

Reply:
Yeah, this object can be undefined

Reply:
Previously we were returning value so we didn't need to check if it's not undefined